content/_guides/channelmodes.md

```diff
@@ -4,25 +4,234 @@ category: chanop
+  allowing you to delegate limited capabilities to other users. 
+- It can maintain channel attributes and perform functions automatically
+  through [control flags](#chanserv-control-flags).
 
+## Channel modes
+
+Channel modes control:
+
+- Bans (`b`, `e`)
+- Moderation (`m`, `n`, `q`, `u`, `v`, `z`)
+- Forwarding (`f`, `F`)
+- Channel visibility (`p`, `s`)
+- Channel invitations (`g`, `i`, `I`)
+- Channel access controls (`j`, `k`, `l`, `n`, `o`, `Q`, `r`, `S`)
+
+### Requirements
+
+- To use `/mode`, you must be a channel operator (`+o`).
+- To use `ChanServ`, your channel must be
+  [registered](/creatingchannels#registering-the-new-channel) and your user
+  flags must allow you to take actions that set modes.
+
+### Mode command summary
+
+| Purpose                    |Command                    |
+|----------------------------|---------------------------|
+| List current channel modes | `/mode <channel>`         |
+| Set a mode                 | `/mode <channel> +<mode>` |
+| Unset a mode               | `/mode <channel> -<mode>` |
+
+### Examples
+
+To allow only operators to set the topic on `#foo`:
+
+```irc
+/mode #foo +t
+```
+
+To set a user `bar` as an operator on `#foo`:
+
+```irc
+/mode #foo +o bar
+```
+
+To ban any user connecting to `#foo` from a given host:
+
+```irc
+/mode #foo +b *!*@10-10-32-1.res.example.com
+```
+
+### Preserving channel modes
+
+By default, all channel modes are lost when a channel becomes empty. To
+preserve modes, set the `ChanServ` `GUARD` [control
+flag](#chanserv-control-flags): 
+
+```irc
+/msg ChanServ SET <channel> GUARD ON
+```
+
+`ChanServ` will join your channel and remain there, which helps prevent the
+channel from being empty, which helps prevent the modes from being lost.
+
+In addition, to prevent modes being lost even if `ChanServ` were to go offline
+and leave the channel empty, set the modes with `MLOCK` as described
+[below](#enforcing-channel-modes).
+
+For more information, see `/msg ChanServ HELP SET GUARD`.
+
+### Enforcing channel modes
+
+`ChanServ` can help ensure modes are always set the way you want for your
+channel ("come hell or high netsplit," to quote `jess`). If someone attempts
+to change the channel modes, `ChanServ` will change them back. 
+
+To set a mode `ChanServ` should enforce, set the `MLOCK` [control
+flag](#chanserv-control-flags):
+
+```irc
+/msg ChanServ SET <channel> MLOCK <mode>
+```
+
+For example, to ensure setting the topic on `#foo` is always restricted to
+operators:
+
+```irc
+/msg ChanServ SET #foo MLOCK +t
+```
+
+With this set, if someone were to attempt `/mode #foo -t`, `ChanServ` would
+immediately revert the change with `/mode #foo +t`.
+
+To list modes currently enforced by `ChanServ`:
+
+```irc
+/msg ChanServ INFO <channel>
+```
+
+### Available channel modes
+
+Channel operators can set the following modes.
+<!-- markdownlint-disable MD013 -->
 {% assign channel_modes = site.data.channelmodes | where_exp:"item","item.restricted != true" %}
```

Comment:
I don't like that these tables get pushed so far down. They're the reference a lot of returning users who already know the commands to send, having to scroll to it is annoying.

This would be solved by moving the above to a separate page or putting the tables first with a short ingress hinting that a deeper explanation will follow.